### PR TITLE
docs(example): Use routes with modals example

### PR DESCRIPTION
**Pull request by juhanakristian**

### CODE REVIEW

File: examples/route-modal/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Route modal
+
+Example of using routes to control visibility of modals.
+
+## Preview
+
+Open this example on [CodeSandbox](https://codesandbox.com):
+
+<!-- TODO: update this link to the path for your example: -->
+
```
Comment (kentcdodds) [inline examples/route-modal/README.md:10]:
```suggestion
```

File: examples/route-modal/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+[![Open in codesandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/route-modal)
+
+## Example
+
+Describe the example and how it demonstrates solving the problem. Reference any relevant files/dependencies if needed.
+
+## Related Links
+
+Link to documentation or other related examples.
```
Comment (kentcdodds) [inline examples/route-modal/README.md:19]: Could you fill these two bits in?

**Review by kentcdodds — CHANGES_REQUESTED** [review]
This one's looking pretty good. I think there's quite a bit of extra stuff in here that is unconnected to the fact that this is a modal though (validation, styling, data model etc.). Is there any way we can tighten this example up a bit?

**Comment by juhanakristian** [comment]
Thanks for the review 👍 I can remove the extra stuff to make the example focused on the modal and route usage.

File: examples/route-modal/README.md (hunk 2 above)
Comment (juhanakristian) [reply examples/route-modal/README.md]: Added a short description and link to React Router v6 Modal example 👍

**Review by kentcdodds — APPROVED** [review]
Super! Thank you!

**Comment by kentcdodds** [comment]
@arnaudambro,

Could you create a reproduction of your attempt and put it in either the discord or a github discussion? It's hard to keep track of comments on PRs and not as many people see them as well so we're limited on the number of people who can help.

**Pull request merged**